aibolit/ast_framework/ast.py

```diff
@@ -140,68 +165,68 @@ def all_children_with_type(self, node: int, child_type: ASTNodeType) -> Iterator
         Resulted list is padded with None to length quantity.
         """
         children_with_type = (
-            child for child in self.tree.succ[node] if self.get_type(child) == child_type
+            child for child in self._tree.succ[node] if self.type(child) == child_type
         )
         children_with_type_padded = chain(children_with_type, repeat(None))
         return list(islice(children_with_type_padded, 0, quantity))
 
     @deprecated(reason='Use ASTNode functionality instead.')
-    def get_binary_operation_name(self, node: int) -> str:
-        assert self.get_type(node) == ASTNodeType.BINARY_OPERATION
+    def binary_operation_name(self, node: int) -> str:
+        assert self.type(node) == ASTNodeType.BINARY_OPERATION
         name_node, = islice(self.children_with_type(node, ASTNodeType.STRING), 1)
-        return self.get_attr(name_node, 'string')
+        return self.attr(name_node, 'string')
 
     @deprecated(reason='Use ASTNode functionality instead.')
-    def get_line_number_from_children(self, node: int) -> int:
-        for child in self.tree.succ[node]:
-            cur_line = self.get_attr(child, 'line')
+    def line_number_from_children(self, node: int) -> int:
+        for child in self._tree.succ[node]:
+            cur_line = self.attr(child, 'line')
             if cur_line is not None:
                 return cur_line
         return 0
 
     @deprecated(reason='Use get_proxy_nodes instead.')
-    def get_nodes(self, type: Union[ASTNodeType, None] = None) -> Iterator[int]:
-        for node in self.tree.nodes:
-            if type is None or self.tree.nodes[node]['node_type'] == type:
+    def nodes(self, type: Union[ASTNodeType, None] = None) -> Iterator[int]:
+        for node in self._tree.nodes:
+            if type is None or self._tree.nodes[node]['node_type'] == type:
                 yield node
 
-    def get_proxy_nodes(self, *types: ASTNodeType) -> Iterator[ASTNode]:
-        for node in self.tree.nodes:
-            if len(types) == 0 or self.tree.nodes[node]['node_type'] in types:
-                yield ASTNode(self.tree, node)
+    def proxy_nodes(self, *types: ASTNodeType) -> Iterator[ASTNode]:
+        for node in self._tree.nodes:
+            if len(types) == 0 or self._tree.nodes[node]['node_type'] in types:
+                yield ASTNode(self._tree, node)
 
     @deprecated(reason='Use ASTNode functionality instead.')
-    def get_attr(self, node: int, attr_name: str, default_value: Any = None) -> Any:
-        return self.tree.nodes[node].get(attr_name, default_value)
+    def attr(self, node: int, attr_name: str, default_value: Any = None) -> Any:
+        return self._tree.nodes[node].get(attr_name, default_value)
 
     @deprecated(reason='Use ASTNode functionality instead.')
-    def get_type(self, node: int) -> ASTNodeType:
-        return self.get_attr(node, 'node_type')
+    def type(self, node: int) -> ASTNodeType:
```

Comment:
It shadows the built-in `type`. Not that it will be a problem here, but sometimes it results in unexpected behavior.